pkg/sdn/plugin/node.go

```diff
@@ -272,11 +272,13 @@ func (node *OsdnNode) Start() error {
 				continue
 			}
 			if vnid, err := node.policy.GetVNID(p.Namespace); err == nil {
-				node.policy.RefVNID(vnid)
+				node.policy.EnsureVNIDRules(vnid)
 			}
 		}
 	}
 
+	go kwait.Forever(node.syncVNIDRules, 24*time.Hour)
```

Comment:
Hm... no... the fact that they're "stale" means there are no pods with those VNIDs on this node, so the rules will never match.

And if the NetNamespace was deleted and a new NetNamespace was created with the same VNID, the old rules would get deleted when the NetNamespace was deleted, so that couldn't be a problem either.